lib/simulator-xcode-9.js

```diff
@@ -583,9 +595,21 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     }
 
     if (globalPrefs.AppleLanguages) {
-      await B.all(SERVICES_FOR_TRANSLATION.map((arg) => this.simctl.spawnProcess([
-        'launchctl', 'stop', arg
-      ])));
+      if (previousAppleLanguages && previousAppleLanguages === globalPrefs.AppleLanguages) {
```

Comment:
Excuse me, I guess we should compare in string like 
`if (previousAppleLanguages && JSON.stringify(previousAppleLanguages) === JSON.stringify(globalPrefs.AppleLanguages)) {` because
I saw a WDA session was killed as [this gist](https://gist.github.com/mwakizaka/ab8bd5179aaed43be4b4cbd8453435d0) when I try `create session` -> `delete session` -> `create session`.

Reply:
It looks like `_.isEqual(previousAppleLanguages, globalPrefs.AppleLanguages)`

Reply:
thanks, fixed